test/c3/appframework/appframework_test.go

```diff
@@ -106,6 +106,11 @@ var _ = Describe("c3appfw test", func() {
 			// Verify RF SF is met
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
 
+			// Verify bundle push status
```

Comment:
It makes sense to check bundle push after verifying app download by init-container. Please re-order the verification.